### ISSUE TITLE
Can't install or activate plugins.

### ISSUE THREAD

**Issue by thomasbeauvais** [shyim/shopware#11]
We are attempting to install the PayPal and Stripe plugins. In production, we get an error when trying to install the PayPal plugin (attached).

In Stage, we have managed to install it with a clean installation (where we can't do this with production) and it appears to be installed. There is a different problem here, when we click "Activate" it appears to work, then when refresh a few minutes later it is "Deactivated" again. It will not stay activated.

There is almost no information in the logs. We have done this on a running system (Prod) and a completely fresh system (Stage).

I can't think of anything that would be different running in a container. 

Any insight would be extremely appreciated here. I am at a loss.

![Screenshot 2020-11-23 at 10 27 20](https://user-images.githubusercontent.com/1192386/99947093-bbcc0400-2d77-11eb-86fc-b061fddcb2dc.png)

![Screenshot 2020-11-23 at 12 07 34](https://user-images.githubusercontent.com/1192386/99955946-744c7480-2d85-11eb-96ca-01dcad1542c1.png)

![Screenshot 2020-11-23 at 12 07 38](https://user-images.githubusercontent.com/1192386/99955950-76163800-2d85-11eb-8afa-44e26268da7b.png)

**Comment by thomasbeauvais** [comment]
Another interesting thing is when we try to "Configure" the PayPal plugin in the "First Run Wizard" where it would just hang.

When manually installed in an Ubuntu 18 Linux, everything works just fine. 

Do you have an idea of why there would be a difference?

**Comment by shyim** [comment]
`custom/plugins` is shared between all containers?

**Comment by thomasbeauvais** [comment]
Right, there is only a single node now. The volume looks complete and has all of the files that the manually installed version has. It's strange.

**Comment by thomasbeauvais** [comment]
When I try to install the plugin manually by logging into the container, it hangs for 10 minutes with no error. 

```
Shopware Plugin Lifecycle Service
=================================

 Activate 1 plugin(s):
 * PayPal Products for Shopware 6 (v2.0.0)

06:13:26 WARNING   [php] Warning: rmdir(/var/www/html/public/theme/e7371aa93dc4ca457de2ab956cfe3cec/assets/icon/default): Directory not empty ["exception" => ErrorException { …}]
06:13:36 WARNING   [php] Warning: rmdir(/var/www/html/public/theme/e7371aa93dc4ca457de2ab956cfe3cec/assets/icon/solid): Directory not empty ["exception" => ErrorException { …}]
06:13:36 WARNING   [php] Warning: rmdir(/var/www/html/public/theme/e7371aa93dc4ca457de2ab956cfe3cec/assets/icon): Directory not empty ["exception" => ErrorException { …}]
06:13:37 WARNING   [php] Warning: rmdir(/var/www/html/public/theme/e7371aa93dc4ca457de2ab956cfe3cec/assets): Directory not empty ["exception" => ErrorException { …}]
06:13:38 WARNING   [php] Warning: rmdir(/var/www/html/public/theme/e7371aa93dc4ca457de2ab956cfe3cec): Directory not empty ["exception" => ErrorException { …}]
```

We are sooooo close to a production running system here. 

What do you need in order to debug this?

**Comment by thomasbeauvais** [comment]
It turns out that the volumes weren't fast enough or something. The logs in Shopware are hard to follow at times to see what the outcome should be. Also, Azure is pretty terrible and not transparent about how their infrastructure might affect certain scenarios. Perhaps I should do a walk through on how to setup Shopware with your Docker image.
